README.md

```diff
@@ -1389,6 +1390,7 @@ Table of Contents
   * [smsreceivefree.com](https://smsreceivefree.com/) — Provides free temporary and disposable phone numbers.
   * [simplebackups.io](https://simplebackups.io/) — Backup automation service for servers and databases (MySQL, PostgreSQL, MongoDB) stored directly into cloud storage providers (AWS, DigitalOcean, Backblaze...). Provides free plan for 1 backup.
   * [SnapShooter](https://snapshooter.com/) — Backup solution for DigitalOcean, AWS, LightSail, Hetzner and Exoscale, with support for direct database, file system and application backups to s3 based storage. Provides a free plan with daily backups for one resource.
+  * [thedev.id](https://thedev.id) — A free domain for developers.
```

Comment:
Isn't it a subdomain?(not ripienaar, but thought to put it up here)

Reply:
Yeah it's a thedev.id subdomain. I'll update it to make it clearer.